e2etest/utils.go

```diff
@@ -1,11 +1,16 @@
 package e2e
 
 import (
+	"context"
 	"fmt"
 	"os/exec"
 
+	pkgerrors "github.com/pkg/errors"
```

Comment:
Why you need to alias it? `errors` is generally intended to be this one, I'd alias anything else conflicts with this.